backend/app/services/device/session_service.py

```
@@ -128,3 +132,89 @@ def _normalize_ttl(self, ttl_seconds: Any) -> int:
+
+    parsed = urlsplit(url)
+    if parsed.hostname not in {"localhost", "127.0.0.1", "::1"}:
+        return result
+
+    sandbox_id = (spec.get("cloudConfig") or {}).get("sandboxId")
+    if not sandbox_id:
+        return result
+
+    try:
+        vm_status = await _get_cloud_device_provider().get_vm_status(sandbox_id)
+    except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)